topdown/errors.go

```diff
@@ -29,6 +29,7 @@ type Error struct {
 	Code     string        `json:"code"`
 	Message  string        `json:"message"`
 	Location *ast.Location `json:"location,omitempty"`
+	Err      error         `json:"-"`
```

Comment:
Can we make this non-public? Looks like we need it from the rego package, but we could also add
```go
func (t *Error) Wrap(e err) *Error {
  t.err = e
  return t
}
```
and use that from outside of the topdown package 🤔

Reply:
Thanks for reviewing. I updated my changes per your suggestion.